README.md

```diff
@@ -0,0 +1,70 @@
+# ERIVE.delivery Shipping Method
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [License](#license)
+- [Support](#support)
+
+## Introduction
+
+This module adds a custom online shipping method named ERIVE.delivery.
```

Comment:
Is it really created? or do you need to assign it in the configuration settings?

Reply:
Yeah, you're right. I just replaced 'Magenta' with 'Shopware' and didn't pay attention to the text.
Read through the whole file, everything seems correct